packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -407,13 +497,16 @@ export default class Meeting extends StatelessSparkPlugin {
     this.resourceId = options.resourceId;
     this.setCorrelationId(this.id);
     // this.state.join();
+    this.spark.meetings.notifyRinging(this, OUTGOING);
     return MeetingUtil.joinMeeting(this)
       .then((response) => {
-        this.setLocusJoin(response);
+        this.setLocus(response);
+        this.spark.meetings.notifyRinging(this, null, false);
```

Comment:
I'm not sure how I feel about calling the plugin internally. I think this might be a better case to either use the event emitter or possibly restructure things. Could you emit the notify ringing event inside of `meeting` instead `meetings` since the ring should only be specific to a particular meeting?

Reply:
Yeah I thought about this and wasn't 100%, but was what I could come up with.
Here's the reasoning for doing it on meetings: It's an incoming 1:1 call, client doesn't know for sure which meeting object to listen to. However, we could maybe change the synchronization/sequence of it so that it's for sure AFTER the meeting_added event occurs and then emit the event directly from the meeting object. restructuring in that way would make logical sense to me (though this does as well) what other options for restructuring were you thinking of?

Reply:
I like your suggestion. Mainly because we _have_ to create a meeting before they can act on it, so it seems like we should create the meeting and then immediately emit the notify ringing (possibly with a flag on the create method?). 

I wasn't thinking about the incoming case so I don't see another way other than what you proposed above. It's possible to merge this since it's so meaty and then pick that one peace up separately.